### PR TITLE
Updated Accepted and Rejected EIPs from ACD call

### PR DESCRIPTION
**Pull request by MadeofTin**
When opening a pull request to submit a new EIP, please use the suggested template: https://github.com/ethereum/EIPs/blob/master/eip-template.md

We have a GitHub bot that automatically merges some PRs. It will merge yours immediately if certain criteria are met:

 - The PR edits only existing draft PRs.
 - The build passes.
 - Your Github username or email address is listed in the 'author' header of all affected PRs, inside <triangular brackets>.
 - If matching on email address, the email address is the one publicly listed on your GitHub profile.

### CODE REVIEW

File: EIPS/eip-1679.md [hunk 1]
```diff
@@ -24,50 +24,49 @@ This meta-EIP specifies the changes included in the Ethereum hardfork named Ista
...
-- TBD
+- [EIP-615](https://eips.ethereum.org/EIPS/eip-615): Subroutines and Static Jumps for the EVM - *Withdrawn*
+- [EIP-663](https://eips.ethereum.org/EIPS/eip-663): Unlimited SWAP and DUP instructions
+- [EIP-1109](https://eips.ethereum.org/EIPS/eip-1109): PRECOMPILEDCALL opcode (Remove CALL costs for precompiled contracts)
+  - requirement of EIP-1962
+- [EIP-1559](https://eips.ethereum.org/EIPS/eip-1559): Fee market change for ETH 1.0 chain
+- [EIP-1930](https://eips.ethereum.org/EIPS/eip-1930): CALLs with strict gas semantic. Revert if not enough gas available.
+- [EIP-1959](https://eips.ethereum.org/EIPS/eip-1959): New Opcode to check if a chainID is part of the history of chainIDs
+- [EIP-1965](https://eips.ethereum.org/EIPS/eip-1965): Method to check if a chainID is valid at a specific block Number
+- [EIP-1985](https://eips.ethereum.org/EIPS/eip-1985): Sane limits for certain EVM parameters
```
Comment (axic) [inline EIPS/eip-1679.md]: I don't think this was rejected.

**Comment by eip-automerger** [comment]
Hi! I'm a bot, and I wanted to automerge your PR, but couldn't because of the following issue(s):

 - EIP 1679 requires approval from one of (@5chdn, @axic)

File: EIPS/eip-1679.md (hunk 1 above)
Comment (MadeofTin) [reply EIPS/eip-1679.md]: Removed from Rejected

File: EIPS/eip-1679.md [hunk 2]
```diff
@@ -24,50 +24,49 @@ This meta-EIP specifies the changes included in the Ethereum hardfork named Ista
...
+- [EIP-2014](https://eips.ethereum.org/EIPS/eip-2014): Extended State Oracle
```
Comment (axic) [inline EIPS/eip-1679.md]: Withdrawn.

File: EIPS/eip-1679.md [hunk 3]
```diff
@@ -24,50 +24,49 @@ This meta-EIP specifies the changes included in the Ethereum hardfork named Ista
...
+- [EIP-2025](https://eips.ethereum.org/EIPS/eip-2025): Funding Eth1.x with Developer Block Rewards *Withdrawn*
+- [EIP-2026](https://eips.ethereum.org/EIPS/eip-2026): State Rent H - Fixed Prepayment for accounts
```
Comment (axic) [inline EIPS/eip-1679.md]: I think all the state rent ones were withdrawn?

File: EIPS/eip-1679.md (hunk 2 above)
Comment (MadeofTin) [reply EIPS/eip-1679.md]: Updated

File: EIPS/eip-1679.md (hunk 3 above)
Comment (MadeofTin) [reply EIPS/eip-1679.md]: Just the one had a reference implementation. Unless that happened on the call after I dropped off.

Comment (MadeofTin) [reply EIPS/eip-1679.md]: All but one were withdrawn

File: EIPS/eip-1679.md [hunk 4]
```diff
@@ -24,50 +24,49 @@ This meta-EIP specifies the changes included in the Ethereum hardfork named Ista
...
-- [EIP-1109](https://eips.ethereum.org/EIPS/eip-1109): PRECOMPILEDCALL opcode (Remove CALL costs for precompiled contracts)
-  - requirement of EIP-1962
 - [EIP-1283](https://eips.ethereum.org/EIPS/eip-1283): Net gas metering for SSTORE without dirty maps
 - [EIP-1344](https://eips.ethereum.org/EIPS/eip-1344): Add ChainID opcode
 - [EIP-1352](https://eips.ethereum.org/EIPS/eip-1352): Specify restricted address range for precompiles/system contracts
 - [EIP-1380](https://eips.ethereum.org/EIPS/eip-1380): Reduced gas cost for call to self
-- [EIP-1559](https://eips.ethereum.org/EIPS/eip-1559): Fee market change for ETH 1.0 chain
-- [EIP-1965](https://eips.ethereum.org/EIPS/eip-1965): Method to check if a chainID is valid at a specific block Number
-- [EIP-1702](https://eips.ethereum.org/EIPS/eip-1702): Generalized account versioning scheme
 - [EIP-1706](https://eips.ethereum.org/EIPS/eip-1706): Disable SSTORE with gasleft lower than call stipend
```
Comment (axic) [inline EIPS/eip-1679.md]: This was "accepted" together with 1283.

File: EIPS/eip-1679.md [hunk 5]
```diff
@@ -24,50 +24,49 @@ This meta-EIP specifies the changes included in the Ethereum hardfork named Ista
...
 - [EIP-1057](https://eips.ethereum.org/EIPS/eip-1057): ProgPoW, a Programmatic
   Proof-of-Work
   - There is a
     [pending audit](https://medium.com/ethereum-cat-herders/progpow-audit-goals-expectations-75bb902a1f01),
     above and beyond standard security considerations, that should be evaluated
     prior to inclusion.
-- [EIP-1108](https://eips.ethereum.org/EIPS/eip-1108): Reduce alt_bn128 precompile gas costs
-- [EIP-1109](https://eips.ethereum.org/EIPS/eip-1109): PRECOMPILEDCALL opcode (Remove CALL costs for precompiled contracts)
-  - requirement of EIP-1962
 - [EIP-1283](https://eips.ethereum.org/EIPS/eip-1283): Net gas metering for SSTORE without dirty maps
```
Comment (axic) [inline EIPS/eip-1679.md]: Not sure if this was properly accepted, but I think it was tentatively together with 1706 (and 2200).

Comment (MadeofTin) [reply EIPS/eip-1679.md]: I left the tentative yes's out. I can add them to a Tentative yes list, or hopefully they get accepted today.

**Pull request merged**